README.md

```diff
@@ -63,6 +63,15 @@ At time of writing, the databases contain only a health collection as referenced
 
 The connection string for MongoDB Compass is `mongodb+srv://pensano-aura-db-user:<password>@cluster0.sylisri.mongodb.net/`, using the shared password.
 
+## Seeding the database collections
+
+The database collections can be seeded with using the `seedCafes.js` file in the `backend` directory.
+
+`cd backend` to navigate to the backend directory.
+
+`npm run seed:cafes:dev` will seed the `cafes` collection in the `aura_DEV` database.
+`npm run seed:cafes:test` will seed the `cafes` collection in the `aura_TEST` database.
+
```

Comment:
Add documentation for seeding the production database

Reply:
Requested changes have been made.